src/fontra/backends/designspace.py

```diff
@@ -123,77 +126,85 @@
     ("vendorID", "openTypeOS2VendorID"),
 ]
 
-
-ufoInfoPrefix = "ufo.info."
```

Comment:
There is a problem with the changing of the keys: it breaks existing data we have as part of the GS project.

Additionally, I'm not really comfortable with removing the "openType" and "postscript" prefixes.

I also don't understand why some names keep their OTF table prefix ("gaspRangeRecords", "headFlags", etc.) and others don't ("created", "codePageRanges", etc.).

Reply:
I went back to the original list of `ufoInfoAttributesToRoundTrip`.

Background infos:
Why did I do a new mapping attributes mapping: I tried to make a similar to `fontInfoNameMapping`. 
There we have mappings like:` ("vendorID", "openTypeOS2VendorID"),`, but I was not able to do a proper name for all the attributes, for example removing `head` from `headFlags` would be too generic, plus: the whole dictionary was not final, it was up for discussion (where do we want to go). And I got some valid answers from you.

I mark this as resolved as well.